src/components/graph/menus/network-modifications-menu.js

```diff
@@ -0,0 +1,79 @@
+/**
+ * Copyright (c) 2021, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import Menu from '@mui/material/Menu';
+import { useIntl } from 'react-intl';
+import Typography from '@mui/material/Typography';
+import { NestedMenuItem } from 'mui-nested-menu';
+import MenuItem from '@mui/material/MenuItem';
+import ListItemText from '@mui/material/ListItemText';
+
+/**
+ * Menu to select network modification to create
+ * @param {Boolean} open Is the dialog open ?
+ * @param {EventListener} onClose Event to close the dialog
+ * @param onOpenDialog handle the opening of dialogs
+ * @param dialogs the list of dialog
+ * @param position
+ */
+const NetworkModificationMenu = ({ open, onClose, onOpenDialog, dialogs }) => {
```

Comment:
This component's name and its filename do not match : either change the filename to `network-modification-menu.js` (without "s"), or change the component's name to `NetworkModificationsMenu` (with a "s").

Reply:
The component's name is changed to NetworkModificationsMenu